atuin-server/migrations/sqlite/20210425153757_create_users.sql

```diff
@@ -0,0 +1,12 @@
+create table users (
+	id INTEGER primary key,               -- also store our own ID
+	username varchar(32) not null unique,   -- being able to contact users is useful
+	email varchar(128) not null unique,     -- being able to contact users is useful
+	password varchar(128) not null unique,
+  created_at timestamp not null default CURRENT_TIMESTAMP
```

Comment:
totally a nit, but could the indentation and spacing on this one be fixed please? just to make it in line with the rest 🙏 

should probably get a formatter for sql.